src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        this.radius = radius;
+        setColor(color);
+    }
+
+    public double calculateArea() {
```

Comment:
While @Override annotation is optional it's nice to put it to separate methods inherited from parent class or interface from own class methods.
Please add everywhere where applicable